templates/init/functions/index.js

```diff
@@ -1,8 +1,8 @@
 const functions = require('firebase-functions');
 
-// // Create and Deploy Your First Cloud Functions
-// // https://firebase.google.com/docs/functions/write-firebase-functions
-//
-// exports.helloWorld = functions.https.onRequest((request, response) => {
-//  response.send("Hello from Firebase!");
-// });
+// Create and Deploy Your First Cloud Functions
```

Comment:
I'd prefer to keep the double-comments here to make sure people are only deploying things they intend to.